EDDI/EDDI.cs

```diff
@@ -1234,6 +1234,1664 @@ private bool eventStarScanned(StarScannedEvent theEvent)
                 star.stellarclass = theEvent.stellarclass;
                 star.solarmass = theEvent.solarmass;
                 star.solarradius = theEvent.solarradius;
+                star.luminosityclass = theEvent.luminosityclass;
```

Comment:
This is a much bigger change than I expected

Reply:
I don't undestand what you want to say

Reply:
You have added 1660 lines of code. ONE THOUSAND SIX HUNDRED AND SIXTY lines of code. Did you not notice that before you submitted your PR?